Sources/ApolloCodegenLib/Templates/TemplateRenderer.swift

```diff
@@ -145,6 +145,7 @@ private struct ImportStatementTemplate {
     static func template(forConfig config: ReferenceWrapped<ApolloCodegenConfiguration>) -> TemplateString {
       """
       \(ImportStatementTemplate.template)
+      @_exported import enum ApolloAPI.GraphQLNullable
```

Comment:
I'm skeptical this needs to be in each generated file. [This article](https://davedelong.com/blog/2018/01/19/simplifying-swift-framework-development/) makes me think we could get away with a top-level `Exports.swift` file that has this in it and it would apply module-wide.